docs/CaliptraIntegrationSpecification.md

```diff
@@ -358,7 +358,7 @@ It is strongly recommended that these PAUSER registers are either set at integra
 
 ### Programmable registers
 
-Caliptra provides 5 programmable registers that SoC can set at boot time to limit access to the mailbox peripheral. The default PAUSER set by the integration parameter CPTRA\_DEF\_MBOX\_VALID\_PAUSER is valid until all programmable registers are consumed. CPTRA\_MBOX\_VALID\_PAUSER registers become valid once the corresponding lock bit CPTRA\_MBOX\_PAUSER\_LOCK is set.
+Caliptra provides 5 programmable registers that SoC can set at boot time to limit access to the mailbox peripheral. The default PAUSER set by the integration parameter CPTRA\_DEF\_MBOX\_VALID\_PAUSER is valid at all times. CPTRA\_MBOX\_VALID\_PAUSER registers become valid once the corresponding lock bit CPTRA\_MBOX\_PAUSER\_LOCK is set.
```

Comment:
So the register doesn't override the RTL parameter?

Reply:
There is a default valid PAUSER that is always valid. Set by it's own parameter.
The programmable registers are used to enable 5 additional PAUSER values that are valid once locked.